web-app/preprocess/csvToJson.py

```diff
@@ -0,0 +1,19 @@
+import csv
+import json
+
+csvFilePath = '2020_05_london.csv'
+jsonFilePath = '2020_05_london.json'
+
+data = []
+with open(csvFilePath) as csvFile:
+    csvReader = csv.DictReader(csvFile)
+    for rows in csvReader:
+        entry = {}
+        entry['Month'] = rows['Month']
```

Comment:
nit: JSON property names should be camelCase: https://google.github.io/styleguide/jsoncstyleguide.xml#Property_Name_Format